package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ocaml-platform",
   "displayName": "OCaml Platform",
-  "version": "1.17.0",
```

Comment:
I don't know what's the precedent, but shouldn't the version be 1.18.0 given we're bumping vscode engine version?

Reply:
Are there any new features introduced by bumping the engine?

Reply:
There are no new features, just a bug fix, so I thought a point release would be enough.